spec/models/user_spec.rb

```diff
@@ -1,5 +1,15 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject { User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.') }
+
+  before { subject.save }
+
+  it 'return the three latest posts for a user' do
+    expect(User.recent_post(subject.id).count).to eql 0
+  end
+
```

Comment:
- [ ] Please add specs for validation as required, it looks something like this :point_down: 

```ruby
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```